performance-test/requirements.txt

```diff
@@ -0,0 +1,3 @@
+locust==2.24.1
+pydantic>=2.6.1
+har2locust==0.9.1
```

Comment:
Use requirements.in maybe from the start?

Reply:
Good shout, I'll do that